jenkins/ci.opensuse.org/openstack-diskimage-builder.yaml

```diff
@@ -0,0 +1,118 @@
+- job:
+    name: 'openstack-diskimage-builder'
+    project-type: matrix
+
+    triggers:
+      - timed: '@daily'
+
+    concurent: false
+
+    build-discarder:
+    num-to-keep: 10
+    days-to-keep: -1
+
+    parameters:
+      - bool:
+          name: BOOT_DIB_IMAGE
+          default: false
+          description: Boot the generated image using QEMU/KVM.
+
+    axes:
+      - axis:
+          type: user-defined
+          name: opensuse_release
+          values:
+            - 42.1
+            - 13.1
+      - axis:
+          type: user-defined
+          name: suse_element
+          values:
+              - opensuse
+              - opensuse-minimal
+      - axis:
+          type: slave
+          name: slave
+          values:
+            - cloud-cleanvm
+
+    execution-strategy:
+      # We can only use one VM at a time, so serialize all jobs
+      sequential: true
+
+    builders:
+      - shell: |
+          #!/bin/bash
+
+          cat > test-diskimage-builder.sh <<EOF
+          #!/bin/bash
+          set -e
+
+          cleanup() {
+              # Clean up in case the last job was broken
+              rm -rf diskimage-builder dib-utils
+              rm -f *.qcow2 diskimage-builder-boot.log
+          }
+
+          install_tools() {
+              zypper in -y git-core qemu-tools
+              # We are going to boot our image but only if we can do it on the
+              # running host
+              ${BOOT_DIB_IMAGE} && zypper in -y qemu-kvm &> /dev/null || true
+          }
+
+          trap cleanup EXIT
+
+          # Start fresh
+          cleanup
+          # Install necessary tools
+          install_tools
```

Comment:
you don't have any repositories defined at this point, so this fails. I suggest to copy code from here:

https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/qa_openstack.sh#L175